ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_service.proto

```diff
@@ -39,6 +40,7 @@ service CommandService {
   // - ``INVALID_ARGUMENT``: if the payload is malformed or is missing required fields
```

Comment:
What's the ordering of these error codes? They don't seem to line up with the numerical order of the [gRPC error codes](https://grpc.github.io/grpc/core/md_doc_statuscodes.html). I don't expect to align them as part of this PR.

Reply:
I contacted Kamil who documented them initially not that long ago. There is no particular order. I can change this if you'd like.

Reply:
It's fine to change this as part of a different PR -- I'm OK with merging this one as-is.